packages/core/src/BlockNoteEditor.ts

```diff
@@ -116,10 +104,7 @@ const blockNoteTipTapOptions = {
 };
 
 // TODO: make type of BareBlock / Block automatically based on options.blocks
-export class BlockNoteEditor<
-  BareBlock extends BlockTemplate<any, any> = DefaultBlockTypes,
-  Block extends BareBlock & { children: Block[] } = WithChildren<BareBlock>
-> {
+export class BlockNoteEditor<Block extends BaseBlock = BaseBlock> {
```

Comment:
Cleaned up some types here too. I don't think that BareBlock is needed here. Why not just making `children` in `BlockTemplate` optional?